### ISSUE TITLE
Content empty, and a load of error messages

### ISSUE THREAD

**Issue by HWiese1980** [codesandbox/codesandbox-client#3141]
# 🐛 bug report

## Description of the problem
The UI of Codesandbox itself works but the expected output of the code is empty (white box).

## How has this issue affected you? What are you trying to accomplish?
Just trying to load some example code from the Elastic UI git repo.

I'm seeing a bunch of error messages (unfortunately in German, sorry, hope it helps determining the error nevertheless): 

```
Anfrage für Zugriff auf Cookies oder Speicher für "https://stats.g.doubleclick.net/r/collect?v=1&aip=1&t=dc&_r=…_gid=1404184136.1575716714&gjid=1052002987&_v=j79&z=35203990" wurde blockiert, weil sie von einem Element zur Aktivitätenverfolgung (Tracker) stammt und das Blockieren von Seitenelementen aktiviert ist.
ll7lnlpm97
Laden fehlgeschlagen für das <script> mit der Quelle "https://api.getvero.com/api/v2/events/track.json?params=%7B%22event_name%22%3A%22Change%20Settings%22%2C%22api_key%22%3A%22348b8acbc93adcf7c3d647d2abb4f4c22fe880e9%22%2C%22development_mode%22%3Afalse%2C%22identity%22%3A%7B%22id%22%3A%22si7fug30enk%22%2C%22timezone%22%3A1%2C%22userAgent%22%3A%22Mozilla%2F5.0%20(Windows%20NT%2010.0%3B%20Win64%3B%20x64%3B%20rv%3A71.0)%20Gecko%2F20100101%20Firefox%2F71.0%22%2C%22platform%22%3A%22Win32%22%2C%22language%22%3A%22de%22%7D%2C%22data%22%3A%7B%22name%22%3A%22customVSCodeTheme%22%2C%22value%22%3A%22%7B%5Cn%5Ct%5C%22type%5C%22%3A%20%5C%22dark%5C%22%2C%5Cn%5Ct%5C%22colors%5C%22%3A%20%7B%5Cn%5Ct%5Ct%5C%22activityBar.background%5C%22%3A%20%5C%22%231c2022%5C%22%2C%5Cn%5Ct%5Ct%5C%22activityBar.border%5C%22%3A%20%5C%22%23111518%5C%22%2C%5Cn%5Ct%5Ct%5C%22activityBarBadge.background%5C%22%3A%20%5C%22%23c0c0c0%5C%22%2C%5Cn%5Ct%5Ct%5C%22activityBarBadge.foreground%5C%22%3A%20%5C%22%231c2022%5C%22%2C%5Cn%5… ll7lnlpm97:1:1
Fehler beim Laden von 'https://codesandbox.io/static/js/sandbox-startup.ad572403.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Fehler beim Laden von 'https://codesandbox.io/static/js/vendors~app~codemirror-editor~monaco-editor~sandbox.57b631f1.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Fehler beim Laden von 'https://codesandbox.io/static/js/common-sandbox.25d2d35a.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Fehler beim Laden von 'https://codesandbox.io/static/js/vendors~sandbox.34b71679.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Fehler beim Laden von 'https://codesandbox.io/static/js/default~app~embed~sandbox.b16585ca.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Fehler beim Laden von 'https://codesandbox.io/static/js/sandbox.a0840556.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Laden fehlgeschlagen für das <script> mit der Quelle "https://codesandbox.io/static/js/sandbox-startup.ad572403.js". ll7lnlpm97.csb.app:20:1
Fehler beim Laden von 'https://codesandbox.io/static/js/sandbox-startup.ad572403.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Laden fehlgeschlagen für das <script> mit der Quelle "https://codesandbox.io/static/js/vendors~app~codemirror-editor~monaco-editor~sandbox.57b631f1.chunk.js". ll7lnlpm97.csb.app:21:1
Fehler beim Laden von 'https://codesandbox.io/static/js/vendors~app~codemirror-editor~monaco-editor~sandbox.57b631f1.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Laden fehlgeschlagen für das <script> mit der Quelle "https://codesandbox.io/static/js/common-sandbox.25d2d35a.chunk.js". ll7lnlpm97.csb.app:22:1
Fehler beim Laden von 'https://codesandbox.io/static/js/common-sandbox.25d2d35a.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Laden fehlgeschlagen für das <script> mit der Quelle "https://codesandbox.io/static/js/vendors~sandbox.34b71679.chunk.js". ll7lnlpm97.csb.app:23:1
Fehler beim Laden von 'https://codesandbox.io/static/js/vendors~sandbox.34b71679.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Laden fehlgeschlagen für das <script> mit der Quelle "https://codesandbox.io/static/js/default~app~embed~sandbox.b16585ca.chunk.js". ll7lnlpm97.csb.app:24:1
Fehler beim Laden von 'https://codesandbox.io/static/js/default~app~embed~sandbox.b16585ca.chunk.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Laden fehlgeschlagen für das <script> mit der Quelle "https://codesandbox.io/static/js/sandbox.a0840556.js". ll7lnlpm97.csb.app:25:1
Fehler beim Laden von 'https://codesandbox.io/static/js/sandbox.a0840556.js'. Ein ServiceWorker hat eine mit 'Error: Both cache and network failed: "No result returned", "TypeError: NetworkError when attempting to fetch resource."' abgelehnte Promise an FetchEvent.respondWith() übergeben. sandbox-service-worker.js:1:11472
Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration . monaco-linter.4f3ec38a.worker.js:1:1528405
```

### Link to sandbox: [link](https://codesandbox.io/s/ll7lnlpm97) (optional)
This is an example sandbox coming from the original [Elastic UI repository](https://elastic.github.io/eui/#/) (top right corner).
### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | I don't know. The latest version so far (PROD-1575663604-6c25d6afd?)
| Browser          | Firefox 70.0.1 
| Operating System | Windows 10 64bit

**Comment by CompuIves** [comment]
Interesting, this seems like a service worker issue in the sandbox. Does it work after a second refresh? Otherwise it might be necessary to delete the cache of the sandbox in the Chrome devtools. I'll take a look into how it got into this state.

**Comment by HWiese1980** [comment]
Nope, it doesn't work after a second refresh, unfortunately. Emtying the cache does not help either. 

Awesome! Looking forward to the results. If I remember correctly, I've seen this behavior on other sites as well since FF 70 or since it has this new tracker blocking feature... it might be related to that. 

Furthermore I'm using _uMatrix_ to block tracking scripts. However, that is completely (including "hide referer" and "hide noscript" stuff) **deactivated** on codesandbox.io and everything below. That is, codesandbox.io is - at least regarding _uMatrix_ - allowed to do anything it wants (i.e., I'm extensively trusting the website).

**Comment by CompuIves** [comment]
Ah, I see! Thanks for the trust, I appreciate that! Do you also have `csb.app` in the whitelist? The project gets executed on that domain, so that might be the issue.

**Comment by HWiese1980** [comment]
Yes, csb.app is also whitelisted. 

Whatever it's worth, I've just tried it on another computer, a MacBook Pro, using Chrome. I also had uMatrix activated but completely whitelisted everything from codesandbox.io - and it had shown the same behavior (not sure about the error messages though as I did not check for them). However, after I deactivated uMatrix completely (i.e., I deactivated the extension itself) it now works here. 

So my conclusion is, even though a little fuzzy because I did change a lot more than just the extension, it *is* somehow related to uMatrix. There still has to be some interference, even if everything is whitelisted and all additional features of uMatrix are deactivated.
